pyproject.toml

```diff
@@ -47,6 +47,7 @@ addonfactory-splunk-conf-parser-lib = "^0.3.3"
 defusedxml = "^0.7.1"
 Faker = "^13.12.0"
 xmltodict = "^0.13.0"
+xmlschema = "^1.11.3"
```

Comment:
Why did you decide to use `xmlschema`? Current tests are using `etree.parse` from here:

https://github.com/splunk/addonfactory-workflow-requirement-files-unit-tests/blob/80cccd0353431c5358b9ec5545d527b92a93d20f/test_lib/validate_xml.py#L38

Reply:
In the original implementation, `lxml` lib was used to deliver etree. I wanted to avoid adding it because there are a lot of license and dependency issues reported for it in FOSSA. `xmlschema` has some license issues reported but there are false-positives probably (I will confirm with the legal team). `defusedxml` doesn't support schema validation natively as I've checked